wunderio/drupal-project-k8s
# SLT-286: Make gdpr-dump work for Drupal 7

This PR moves the GDPR-dump configuration from a dedicated file to silta.yml, which is then mounted as a mysql configuration file, which works independently of the drush version (drush 8 does not support the --extra-dump parameter).

Benefits:

- less configuration files
- possibility to centrally update default rules
- should work with Drupal 7

To test:
- log in via SSH
- run `drush sql-dump | less` and look for the content of the `users_field_data` table
- usernames and e-mail addresses should be anonymised.
